content/200-concepts/150-data-platform/08-billing/02-plans-and-quotas.mdx

```diff
@@ -0,0 +1,142 @@
+---
+title: 'Plans and quotas'
+metaTitle: 'Plans and quotas'
+metaDescription: 'The Prisma Data Platform provides a set of monthly plans that include quotas for the number of collaborators and environments per project, as well as a quota for the usage of the Prisma Data Proxy (in CPU milliseconds).'
+---
+
+<TopBlock>
+
+The Prisma Data Platform provides a set of monthly plans that include quotas for:
+
+- number of collaborators per project
+- number of environments per project
+- Prisma Data Proxy usage (in CPU milliseconds) per project
+
+</TopBlock>
+
+## About plans and quotas
+
+Plans in the Prisma Data Platform are designed with the goal to help you scale your projects as the need for team members, environments, and Data Proxy usage grows.
+
+Because of this, a single Prisma Data Platform plan applies for a single project.
+
+We want to give you the option to create projects freely and you can then decide which project you want to upgrade to a paid plan. You can have 100 Prisma projects and you can upgrade only 2 of them to paid plan.
```

Comment:
>You can have 100 Prisma projects

This is a global "safety-limit", I am not sure it deserves this level of attention. I would say "create many" or something along those lines. We might decide to lower this limit later. Having the limit spelled out here make this subject to getting stale. If at all, I would mention it in the platform limits page, which at least is one centralized source of truth.

>you can upgrade only 2 of them to paid plan

Is this so? First time I hear. Also, I would not understand the reasoning behind limiting the number of projects people can pay for.

Reply:
Ooops. This was intended as an example, but I clearly made it sound like a true-to-life scenario. "100" is replaced by "many" and 2 is replaced by "several". It sounds vague, but then it's also gives a correct picture (I hope). Let me know if it doesn't work.